polling_stations/apps/data_finder/tests/test_ee_wrapper.py

```diff
@@ -76,7 +77,7 @@ def get_data_with_elections(self, query_url):
             "group_type": None,
             "explanation": "some text",  # explanation key contains text
             "metadata": {
-                "this election": "has some metadata"
+                "2019-05-02-id-pilot": {"this election": "has an id pilot"}
```

Comment:
Caps "ID"

Reply:
all about the grammatically correct test fixtures :D